packages/forms-web-app/src/controllers/selected-appeal/appeal-details/index.js

```diff
@@ -84,3 +84,45 @@ const formatDetailsSuffix = (userType) => {
 	}
 	return "Appellant's";
 };
+
+/**
+ * @param {import('appeals-service-api/src/spec/api-types').AppealCaseWithAppellant} caseData
+ * @param {string} userType
+ * @returns {import("@pins/common/src/view-model-maps/rows/def").Rows}
+ */
+
+const retrieveDetailsRowsForUser = (caseData, userType) => {
+	const rows = detailsRows(caseData);
+
+	if (userType === (APPEAL_USER_ROLES.APPELLANT || APPEAL_USER_ROLES.AGENT)) {
+		rows.push({
+			keyText: 'Award of costs',
+			valueText: 'Yes',
+			condition: (caseData) => caseData.costsAppliedForIndicator
+		});
+	}
+
+	return rows;
+};
```

Comment:
Sorry but not this. Just pass user type into `detailsRows` and work out this display logic with a condition based on caseData and userType, we've already done that somewhere else

Reply:
Changed, should be neater now